client/python/conductor/ConductorWorker.py

```diff
@@ -15,26 +15,26 @@
 from __future__ import print_function, absolute_import
 import sys
 import time
-import subprocess
-import conductor
 from conductor.conductor import WFClientMgr
 from threading import Thread
 import socket
 
 hostname = socket.gethostname()
 
-class ConductorWorker:
-    def __init__(self, server_url, thread_count, polling_interval):
+
+class ConductorWorker(object):
```

Comment:
You don't need to explicitly inherit `object` in python 3, this only applies to python 2
Unless this is fixing a bug, I would leave it out.
This page has a nice explanation: https://stackoverflow.com/questions/4015417/python-class-inherits-object